tree/ntuple/test/CMakeLists.txt

```diff
@@ -78,7 +78,9 @@ if(MSVC)
 endif()
 ROOT_ADD_GTEST(rfield_vector rfield_vector.cxx LIBRARIES ROOTNTuple CustomStruct)
 
-ROOT_ADD_GTEST(ntuple_minifile ntuple_minifile.cxx LIBRARIES ROOTNTuple Physics Tree)
+if(NOT MSVC OR builtin_gtest)
```

Comment:
Why does this test fail on MSVC with an external gtest?

Reply:
There is an undefined symbol, and if I use the proper flag to fix it, many (all?) the other tests fail with symbols already defined... But this is not a big issue, AFAIK the tests are not run in Conda (but I do...)